grails-app/conf/BuildConfig.groovy

```diff
@@ -24,7 +24,7 @@ grails.project.test.reports.dir = "target/test-reports"
 def dm, dmClass
 try {
     dmClass = new GroovyClassLoader().parseClass(
-            new File('../transmart-dev/DependencyManagement.groovy'))
+            new File('../transmart-dev/SmartRDependencyManagement.groovy'))
```

Comment:
Hi Anthony. You probably had to modify transmart-dev or at least add SmartRDependencyManagement.groovy to it right? 
If yes - could you please create PR also against https://github.com/thehyve/transmart-dev ? As this is the origin from which dependency manager is being cloned.